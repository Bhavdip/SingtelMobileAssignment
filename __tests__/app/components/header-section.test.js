import React from 'react';
import { shallow } from 'enzyme';
import HeaderSection from '../../../app/components/header-section';

const setUpComponent = (props = {}) => {
  return shallow(<HeaderSection {...props} />);
};

describe('Test HeaderSection', () => {
  let wrapperComponent = {};
  let props = {};
  beforeEach(()=> {
    wrapperComponent = setUpComponent(props);
  });
  it('Test Snapshot Match', ()=> {
    expect(wrapperComponent).toMatchSnapshot();
  });
});
