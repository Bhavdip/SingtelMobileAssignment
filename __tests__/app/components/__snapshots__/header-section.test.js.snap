// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HeaderSection Test Snapshot Match 1`] = `ShallowWrapper {}`;
